docs/gl_objects/users.rst

```diff
@@ -218,6 +243,10 @@ List only direct group memberships::
 
     memberships = user.memberships.list(type='Namespace')
 
+.. note::
+
+    Administrator access is required.
```

Comment:
Sorry, just noticed - I like the sentence here, but just to be consistent with the other notes about admin access:

```suggestion
    This endpoint requires admin access.
```

Reply:
Done 😊🔥